train.py

```diff
@@ -324,17 +315,4 @@ def train(args):
 
 
 if __name__ == "__main__":
-    args = parse_args()
-
-    # data sync for cloud platform if enabled
-    if args.enable_modelarts:
```

Comment:
Why remove modelarts training support?

Reply:
1. openi changed the name of args. We need to check the new [tutorial](https://openi.pcl.ac.cn/OpenIOSSG/MNIST_Example)
2. We will add a new wrap-based launcher in `scripts`.

so we drop the openi. see motivation.